### DISCUSSION TITLE
Should android project be shown here?

### DISCUSSION THREAD

**Discussion by synthetic-blood** [AvaloniaUI/Avalonia#10881] [Q&A]
I ran dotnet new avalonia.xplat in cmd and then openned the solution file in visual studio 2022 but looks like only desktop, browser and ios are avalible for debug

![Screenshot 2023-04-02 100943](https://user-images.githubusercontent.com/61121208/229355514-7080710e-221a-495f-8da9-5b4de7ab3fa6.png)

**Comment by timunie** [comment]
Probably Visual Studio is buggy. Is the Android project added to the Solution correctly?

**Reply by synthetic-blood** [reply]
yes i think so, look
![Screenshot 2023-04-03 114218](https://user-images.githubusercontent.com/61121208/229544139-19264928-594e-4388-8261-571191a90d46.png)
edit: also tried adding and removing both in visual studio and cmd but the problem still persists

**Reply by maxkatz6** [reply]
Enable project deploy in solution settings
